### ISSUE TITLE
Working directory is not set to root when creating a notebook

### ISSUE THREAD

**Issue by mohammad-sdsc** [SwissDataScienceCenter/renku-notebooks#267]
## Description

In the interactive env, if we go to a sub-folder and then create a notebook, the working directory of the notebook is not set to the root of the home directory (i.e. `/work/<project>`). It would be less confusing if the working directory of the notebooks is always set to the home directory.

## Reproduce

1. Start an interactive env
2. On the file explorer and go to a sub-folder (e.g. `data`)
3. Create a new notebook
4. Running `%pwd` in the notebook shows `/work/<project>/data` as the current directory.

**Comment by emmjab** [comment]
I think @mohammad-sdsc 's request makes sense -- the below looks pretty bad but I end up doing this a lot:
```
   19  cd ~             # assume i'm in like "/home/e.jablonski"
   20  ls               # wait where is my stuff? oh yeah, it's in "work".
   21  cd work/ 
   22  ls               # lol ok so "work" wasn't in ~
   23  cd /home/jovyan  # wait maybe it's in "jovyan home"
   24  ls               # nope.
   25  pwd          
   26  cd /work         # oh yeah, it was in /work
```
When you first get dropped into the notebook you're fine, but afaict if you `cd ~` to get back home from wherever you go, there's no way you'd remember you started out in `/work` besides having noticed you were dropped into `/work` when you started and having a good memory. (maybe that's not the case! is there some other thing i'm missing?)

Why don't we clone into home? is this a jupyter thing?

**Comment by rokroskar** [comment]
I don't think this is what this issue is about, but there is another one that I think is addressing what you're talking about: https://github.com/SwissDataScienceCenter/renkulab-docker/issues/68

**Comment by emmjab** [comment]
Ah yeah, that's true -- thanks!
